bundles/org.openhab.binding.jeelink/src/main/java/org/openhab/binding/jeelink/internal/connection/JeeLinkSerialConnection.java

```diff
@@ -69,9 +70,12 @@ public synchronized void openConnection() {
             if (!open) {
                 logger.debug("Opening serial connection to port {} with baud rate {}...", port, baudRate);
 
-                CommPortIdentifier portIdentifier;
+                SerialPortIdentifier portIdentifier = serialPortManager.getIdentifier(port);
+                if (portIdentifier == null) {
+                    notifyAbort("Port not found: " + port);
+                    return;
+                }
```

Comment:
Is it possible to move this logic into `JeeLinkHandler`'s initialize method so you can put the thing offline with a config error? You should be able to pass around the `SerialPortIdentifier` instance instead of the `SerialPortManager` instance.

Reply:
After revisiting the code it looks like it is possible. The static method was only used by the handler so that saves another level of passing stuff around. :-)